lib/minitest.rb

```diff
@@ -439,8 +440,10 @@ def prerecord klass, name
     end
 
     ##
-    # Record a result and output the Runnable#result_code. Stores the
-    # result of the run if the run did not pass.
+    # Output and record the result of the test. Call
+    # {result#result_code}[rdoc-ref:Runnable#result_code] to get the
```

Comment:
This should link cleanly with just `+result_code+`, no?

Reply:
That just formats it as code.

Reply:
Huh... that used to work in rdoc...

Reply:
Checked again in RDoc 4.3.0 and 6.0.1; no links. The funny thing is Darkfish formats `+result_code+` with `code` tags and gives it this styling: 

```css
main .method-detail {
    margin-bottom: 2.5em;
    cursor: pointer;
}
```

So it still can look appear to be a link but, it's not. 😿